gravitee-apim-repository/gravitee-apim-repository-api/src/main/java/io/gravitee/repository/log/v4/model/analytics/ApiAnalytic.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.repository.log.v4.model.analytics;
+
+import io.gravitee.common.http.HttpMethod;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class ApiAnalytic {
```

Comment:
```suggestion
public class ApiLog {
```
One "analytic" does not really exist in English. Would "log" work? Or if it is a combination of multiple indexes, could it be `ApiEnrichedLog` or something like that?

Reply:
Or `ApiMetricsLog`?